src/node/internal/public_process.ts

```diff
@@ -251,7 +249,7 @@ export function uptime(): number {
 }
 
 export function loadEnvFile(path: string | URL | Buffer = '.env'): void {
-  if (!compatibilityFlags.experimental || !compatibilityFlags.nodejs_compat) {
+  if (!flags.workerdExperimental || !flags.nodeJsCompat) {
```

Comment:
This seems unrelated and unnecessary. why switch from the use of `globalThis.Cloudflare.compatibilityFlags`? If anything we should be replacing the internal uses of the older `workerd:compatibility-flags` with the global.

Reply:
I've commented about this here: https://github.com/cloudflare/workerd/pull/4591#discussion_r2236974629